### ISSUE TITLE
Color Picker in WinUI2.7 Color options should be laid out left to right

### ISSUE THREAD

**Issue by sanderl** [microsoft/microsoft-ui-xaml#5907]
**Describe the bug**

The new color picker UI in the most detailed view, has the resulting color between the color selection options. It would be more clear if the user chose options from left to right and had a resulting color as the last item. 
![image](https://user-images.githubusercontent.com/46726333/133864209-afcd6323-72c8-4cf3-8b2e-84422fb55102.png)

**Steps to reproduce the bug**

1. Select the color in the color picker.
2. The resulting color is impacted by controls on the left and right of the resulting color.
3. 
**Expected behavior**

Users select the color options on the left of the resulting color. So the user would see (starting on the left) the color selector, then the color values control, then the opacity values and then the color preview control.

**Version Info**

WinUI 2.7

NuGet package version: 

Microsoft.UI.Xaml.2.7.0

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
|  |  |

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2021 Update (19043)            |    |
| October 2020 Update (19042)        |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            |  |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

**Comment by chingucoding** [comment]
I think this is by design (and not really bad at all) but that's just my opinion. @gabbybilka FYI

**Comment by StephenLPeters** [comment]
Proposal is to move the color preview all the way to the right.

**Comment by sanderl** [comment]
> Proposal is to move the color preview all the way to the right.

Yes. That is my proposal because then all of the color adjusters are grouped together and they show an output of the preview color.
